hack/arktos-up.sh

```diff
@@ -610,6 +610,12 @@ echo "*******************************************"
 
 print_success
 
+if [ "${CNIPLUGIN}" == "mizar" ]; then
```

Comment:
This change only works for scale up case, please update the title.

Reply:
Added code for both local scale up and local scale out.